python/tests/operations/random_like.py

```diff
@@ -0,0 +1,145 @@
+import os
+import unittest
+
+import numpy as np
+
+import equistore
+
+
+DATA_ROOT = os.path.join(os.path.dirname(__file__), "..", "data")
+
+
+class TestRandomLike(unittest.TestCase):
+    def test_random_uniform_like_nocomponent(self):
+        tensor = equistore.load(
+            os.path.join(DATA_ROOT, "qm7-power-spectrum.npz"),
+            # the npz is using DEFLATE compression, equistore only supports STORED
+            use_numpy=True,
+        )
+        rand_tensor = equistore.random_uniform_like(tensor)
+
+        self.assertTrue(np.all(tensor.keys == rand_tensor.keys))
+        for key, rand_block in rand_tensor:
+            block = tensor.block(key)
+            self.assertTrue(np.all(rand_block.samples == block.samples))
+            self.assertTrue(np.all(rand_block.properties == block.properties))
+            self.assertEqual(len(rand_block.components), len(block.components))
+            self.assertTrue(
+                np.all(
+                    [
+                        np.all(rand_block.components[i] == block.components[i])
+                        for i in range(len(block.components))
+                    ]
+                )
+            )
```

Comment:
I would remove the check of the metadata and use before the loop
`python
assert equistore.equal(rand_tensor,tensor,only_metadata=True)
`
Like this it checks if the metadata are the same, then you can leave the loop to checks the values

Reply:
same thing for the gradinet, just leave the check of the values

Reply:
Is a gradinet a small gradient 😁 ? 🇮🇹 🤌

Reply:
it is a gradient "gratinato"